doc/TILE_ON_FRAME_BOUNDARY.md

```diff
@@ -0,0 +1,35 @@
+# Tile on a Frame boundary
+
+## Accessing Tiled-pixels outside of a frame
+
+Tiled input and output planes (type PlaneRegion) are configured in order to
+allow memory access of its pixels that are located outside of frame boundary.
+This configuration is only effective when the tile is adjacent to right or bottom frame boundary and
+a partition block straddles on any frame boundary.
+
+The type names of tiled input and output in the codebase is defined in:
+
+"'TileStateMut' struct"
```

Comment:
Nit: Can we use code and code blocks for this part so it looks nicer.

Reply:
Sur- Is ``` for code block?